tests/unit/test_main.py

```diff
@@ -3,6 +3,8 @@
 
 @mock.patch("cou.cli.entrypoint")
 def test_init(cli):
-    from cou import __main__  # noqa: F401
+    import importlib
```

Comment:
not-blocking: There is no need import build in lib inside the test function, so it can be moved to top of the file.

Reply:
I would like to emphasize that I am using this just for this function. But it does not seem right to me either. Fixed.

Reply:
This could be used if we have some singleton object in cou or similar patter, which could break all test due some import error. However only for our cou package, since we are not testing functionality of importlib.